### PR TITLE
Cpu command update

### PR DESCRIPTION
**Pull request by rminnich**

### CODE REVIEW

**Review by brho — APPROVED** [review]

File: cmds/cpu/cpu9p_unix.go [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+	// 	// They had 255 possible values, and use 3, ... ay yi yi.
+	// 	// 9P2000.L doesn't really fit the spirit of 9P.
+	// 	err = p9.ELockError
+	// }
+	switch locktype {
+	case 0, 1:
+		if err := unix.Flock(int(l.file.Fd()), unix.LOCK_EX); err != nil {
+			err = syscall.Errno(2)
+		}
+	case 2:
```
Comment (brho) [inline cmds/cpu/cpu9p_unix.go]: what are the locktype values?  0-> read, 1->write, 2->unlock?  (based on the commented out stuff)

 it looks like you have the same code for case 2 as for 0 and 1.  if 2 == unlock, i'd have expected unix.Funlock or something.

**Review by brho — APPROVED** [review]

**Review by brho — APPROVED** [review]

**Comment by brho** [comment]
caveats to my review: i'm barely go literate, and i keep trying to approve each commit individually.  hence the multiple 'approvals'.

File: cmds/cpu/cpu9p_unix.go (hunk 1 above)
Comment (rminnich) [reply cmds/cpu/cpu9p_unix.go]: yeah that's a screwup on my part, will fix, good catch.

**Comment by rminnich** [comment]
I won't be pushing this until the 9p changes clock in, but thanks for the review :-)

**Pull request merged**